docker/resource-provider/Dockerfile

```diff
@@ -28,7 +24,17 @@ COPY --from=golang:1.22.4-alpine /usr/local/go/ /usr/local/go/
 ENV PATH="/usr/local/go/bin:${PATH}"
 
 COPY . .
-RUN go build -v .
+
+FROM base AS build-gpu
+RUN nvcc --version &&  nvcc --ptx -o ./pkg/resourceprovider/cudaminer/keccak.ptx  ./pkg/resourceprovider/cudaminer/keccak.cu
+RUN go build -v --tags cuda
+ENV DISABLE_POW=false
```

Comment:
Why did we need to set the DISABLE_POW here?

Reply:
This gets passed to `lilypad resource-provider` as `--disable-pow` (we disable pow for CPU ... i.e. local dev / integration tests)